code/__DEFINES/bodyparts.dm

```diff
@@ -45,9 +45,15 @@
 #define AUGGED_HEAD_EMP_GLITCH_DURATION 6 SECONDS
 
 // Color priorities for bodyparts
-#define LIMB_COLOR_HULK 10
-#define LIMB_COLOR_CARP_INFUSION 20
+#define LIMB_COLOR_HULK 1
+#define LIMB_COLOR_CARP_INFUSION 2
+#define LIMB_COLOR_CS_SOURCE_SUICIDE 3
 /// Base priority for atom colors, gets atom priorities added to it
-#define LIMB_COLOR_ATOM_COLOR 30
+#define LIMB_COLOR_ATOM_COLOR 4
 /// Voidwalker effect color priority
-#define LIMB_COLOR_VOIDWALKER_CURSE 40
+#define LIMB_COLOR_VOIDWALKER_CURSE 5
```

Comment:
I would suggest leaving numbers between these values to leave room to grow. (I know the green abductor pr will conflict with this)